.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Is this expected? why so red?

Reply:
Yeah I think these comments are usually not allowed in JSON, hence why that github syntax highlighter doesn't like it... but it works fine in vscode (this is in itself generated by vscode).